app/Website/src/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Website\Http\Requests\ContactFormRequest;
+use Knuckles\Scribe\Attributes\BodyParam;
+use Knuckles\Scribe\Attributes\Endpoint;
+use App\Website\Mail\ContactMail;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Mail;
+
+class ContactController extends Controller
+{
+    #[Endpoint(title: "Submit Contact Form", description: "This endpoint is used for submitting contact form data.")]
+    #[BodyParam(name: "first_name", type: "string", description: "The first name of the user.", required: true)]
+    #[BodyParam(name: "last_name", type: "string", description: "The last name of the user.", required: true)]
+    #[BodyParam(name: "email", type: "string", description: "The email address of the user.", required: true)]
+    #[BodyParam(name: "subject", type: "string", description: "The subject of the message.", required: true)]
+    #[BodyParam(name: "message", type: "string", description: "The body of the message.", required: true)]
+    public function sendContactEmail(ContactFormRequest $request): \Illuminate\Http\JsonResponse
+    {
+        try {
+            $mailData = $request->only(['first_name', 'last_name', 'email', 'subject', 'message']);
```

Comment:
As this is form request, you can access all validated fields with `$request->validated()`, to simplify this.